contracts/README.md

```diff
@@ -0,0 +1,85 @@
+# Contract Addresses
+
+## SEPOLIA (Chain ID: 11155111)
+
+- INTENT_FACTORY: [0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F](https://sepolia.etherscan.io/address/0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F)
+- GROTH16_VERIFYER: [0x4660483e004e416D41bfd77D6425e98543beB6Ba](https://sepolia.etherscan.io/address/0x4660483e004e416D41bfd77D6425e98543beB6Ba)
+- MOCK_USDT: [0x52247fe50A1c11773B182Afd1Dda181de705289c](https://sepolia.etherscan.io/address/0x52247fe50A1c11773B182Afd1Dda181de705289c)
+
+## OP STACK ROLLUP (Chain ID: 357)
+
+- INTENT_FACTORY: [0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F](https://explorer-jam-ccw030wxbz.t.conduit.xyz/address/0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F)
+- GROTH16_VERIFYER: [0x4660483e004e416D41bfd77D6425e98543beB6Ba](https://explorer-jam-ccw030wxbz.t.conduit.xyz/address/0x4660483e004e416D41bfd77D6425e98543beB6Ba)
+- MOCK_USDT: [0x52247fe50A1c11773B182Afd1Dda181de705289c](https://explorer-jam-ccw030wxbz.t.conduit.xyz/address/0x52247fe50A1c11773B182Afd1Dda181de705289c)
+
+## INK SEPOLIA (Chain ID: 763373)
+
+- INTENT_FACTORY: [0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F](https://explorer-sepolia.inkonchain.com/address/0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F)
```

Comment:
Can you have this verified
 https://explorer-sepolia.inkonchain.com/address/0xb1A522F471b48e0bB418c14b7B9C62C746c4Cf5F?tab=contract